it/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -190,229 +190,229 @@ sendAnyway=Send anyway
+12570=Errore nell'allegare %S. Verifica se hai accesso al file.
 
 ## @name NS_MSG_ERROR_DOING_FCC
-12571=Errore durante la copia del messaggio nella cartella 'Posta inviata'. Riprovare?
+12571=Si è verificato un errore durante la copia del messaggio nella cartella 'Posta inviata'. Riprovare?
 
 ## @name NS_ERROR_SMTP_GREETING
-12572=Si è verificato un errore durante l'invio della posta: il server di posta ha inviato una stringa non corretta %s.
+12572=Si è verificato un errore durante l'invio della e-mail: il server di posta ha inviato una risposta non corretta %s.
 
 ## @name NS_ERROR_SENDING_RCPT_COMMAND
-12575=Si è verificato un errore durante l'invio della posta. Il server di posta ha risposto: %1$s. Controllare il destinatario del messaggio %2$s e riprovare.
+12575=Si è verificato un errore durante l'invio della e-mail. Il server di posta ha risposto: %1$s. Verifica il destinatario del messaggio %2$s e riprova.
 
 ## @name NS_ERROR_COULD_NOT_LOGIN_TO_SMTP_SERVER_AUTH_NONE
-12581=Errore durante l'invio della posta: impossibile autenticarsi presso il server SMTP %S. Il server non consente alcun meccanismo di autenticazione (SMTP-AUTH). Togliere la spunta da "Usa nome utente e password" per quel server o contattare il proprio service provider.
+12581=Si è verificato un errore durante l'invio della e-mail: impossibile autenticarsi presso il server SMTP %S. Il server non supporta l'autenticazione (SMTP-AUTH), ma hai scelto ugualmente di utilizzarla. Togli la spunta a "Usa nome utente e password" per il server o contatta il tuo fornitore di servizi.
 
 ## @name NS_ERROR_STARTTLS_FAILED_EHLO_STARTTLS
-12582=Errore durante l'invio della posta: impossibile stabilire un collegamento sicuro col server SMTP %S usando STARTTLS in quanto non supporta tale caratteristica. Disattivare STARTTLS per quel server o contattare il proprio service provider.
+12582=Si è verificato un errore durante l'invio della e-mail: impossibile stabilire un collegamento sicuro col server SMTP %S utilizzando STARTTLS in quanto non supporta tale caratteristica. Disattiva STARTTLS per il server o contatta il tuo fornitore di servizi.
 
 ## @name NS_ERROR_SMTP_PASSWORD_UNDEFINED
 12584=Si è verificato un errore durante l'invio della posta: impossibile ottenere la password per %S. Il messaggio non è stato inviato.
 
 ## @name NS_SMTP_PASSWORD_PROMPT_TITLE
-12585=SMTP Server Password Required
+12585=Password richiesta per server SMTP
 
 ## @name NS_ERROR_SMTP_TEMP_SIZE_EXCEEDED
-12586=La dimensione del messaggio che si cerca di inviare eccede temporaneamente il limite massimo del server. Il messaggio non è stato inviato; tentare di ridurre la dimensione del messaggio o aspettare un po' di tempo e riprovare. Il server ha risposto: %s.
+12586=Le dimensioni del messaggio che stai cercando di inviare superano un limite temporaneo del server. Il messaggio non è stato inviato; cerca di ridurre le dimensioni del messaggio o attendi e riprova. Il server ha risposto: %s.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+12586=Le dimensioni del messaggio che stai cercando di inviare superano il limite temporaneo del server. Il messaggio non è stato inviato; cerca di ridurre le dimensioni del messaggio o attendi e riprova. Il server ha risposto: %s.